libs/community/langchain_community/retrievers/milvus.py

```diff
@@ -59,8 +59,8 @@ def _get_relevant_documents(
         run_manager: CallbackManagerForRetrieverRun,
         **kwargs: Any,
     ) -> List[Document]:
-        return self.retriever.get_relevant_documents(
```

Comment:
is there any magic with callbacks that would make these the same for a BaseRetriever?
Case 1:
```python
self._get_relevant_documents(query, run_manager=run_manager.get_child())
```
Case 2:
```python
callback_manager = CallbackManager.configure(callbacks=run_manager.get_child(), ...)
run_manager = callback_manager.on_retriever_start(...)
result = self._get_relevant_documents(query)
```

Reply:
Can call it in `def get_relevant_documents()` probably if `run_manager` is passed in as a kwarg? Seems similar to how we do it in chat model token generation

Reply:
Sorry - in *invoke

Reply:
ah makes sense thank you!